style.css

```diff
@@ -0,0 +1,624 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans&display=swap');
+
+
+
+div,section{
+    display: block;
+}
+
+body{
+    font-size: 1rem;
+    font-family: arial,sans-serif;
+    color: #222;
+    padding: 0;
+    margin: 0;
+}
+
+.headGoogle{
+    width: 50%;
+    margin: 0 auto;
+    font-family: 'Kumbh Sans', sans-serif;
+    text-align: center;
+    font-size: 2.3rem;
+    margin-top: 2.5em;
+}
+
+.googleimg{
+    aspect-ratio: auto 272 / 92;
+    height: 92px;
+    width: 272px;  
+}
+
+/* .gcolor{
+    color: #4285F4;
+}
+
+.firsto{
+    color: #EA4335;
+}
+
+.secondo{
+    color: #FBBC06;
+}
+
+.lcolor {
+    color: #34A853;
+} */
```

Comment:
Same as the previous comment remove unused code from your codebase, to make your code clean.